package.json

```
@@ -844,7 +844,7 @@
                     "when": "false"
                 },
                 {
-                    "command": "aws.copyLogStreamName",
+                    "command": "aws.copyLogResource",
```

nit: let's keep "name" in the command name since that is clearer and matches many existing similar commands
```suggestion
                    "command": "aws.copyLogName",
```